specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -7,6 +7,54 @@
   },
   "paths": {},
   "definitions": {
+    "AcsIncomingCallEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Communication.IncomingCall event",
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The communication identifier of the target user.",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2022-07-13/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "from": {
+          "description": "The communication identifier of the user who initiated the call.",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2022-07-13/common.json#/definitions/CommunicationIdentifierModel"
+        },
+        "serverCallId": {
+          "description": "The Id of the server call",
+          "type": "string"
+        },
+        "callerDisplayName": {
+          "description": "Display name of caller.",
+          "type": "string"
+        },
+        "customContext": {
+          "description": "Custom Context of Incoming Call",
+          "type": "object",
```

Comment:
Please add a definition for this type.

Reply:
I dont understand

Reply:
This needs to be its own definition, e.g. https://github.com/Azure/azure-rest-api-specs/pull/23762/files#diff-4b49e91c6d3dc4a0bf20909734cce568db54e4721a79a675e92c1dcb6440c53eR858

Reply:
thats what i had initailly but was told to add it here. changing back